src/sqlfluff/cli/commands.py

```diff
@@ -152,16 +152,18 @@ def common_options(f: Callable) -> Callable:
         count=True,
         default=None,
         help=(
-            "Verbosity, how detailed should the output be. This is *stackable*, so `-vv`"
-            " is more verbose than `-v`. For the most verbose option try `-vvvv` or `-vvvvv`."
+            "Verbosity, how detailed should the output be. This is *stackable*, so "
+            "`-vv` is more verbose than `-v`. For the most verbose option try `-vvvv` "
+            "or `-vvvvv`."
         ),
     )(f)
     f = click.option(
         "-n",
         "--nocolor",
         is_flag=True,
         default=None,
-        help="No color - if this is set then the output will be without ANSI color codes.",
+        help="No color - if this is set then the output will be without ANSI color "
+        "codes.",
```

Comment:
```suggestion
        help=(
            "No color - if this is set then the output "
            "will be without ANSI color codes."
        ),
```
Use brackets when splitting strings over mulitple lines.

Reply:
Better suggestions - make it fit :-)

```suggestion
        help="No color - output will be without ANSI color codes.",
```